packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -262,8 +265,37 @@ export default function CollabSidebar() {
 		};
 	}, [] );
 
+	const [ blocks ] = useEntityBlockEditor( 'postType', postType, {
+		id: postId,
+	} );
+
+	const getCommentIdsFromBlocks = () => {
```

Comment:
Let's use named functions?

Reply:
Also, can't this function be moved out of the component? It doesn't seem to depend on other variables.